cli/cmd/cluster-apply.go

```diff
@@ -85,6 +85,17 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		ctxLogger.Fatalf("Verify cluster: %v", err)
 	}
 
+	// Update kube-system namespace with labels. Only for non-managed platforms.
+	if !p.Meta().Managed {
+		labels := map[string]string{
+			"lokomotive.kinvolk.io/name": "kube-system",
```

Comment:
It will, as controlplane will be only running in `kube-system` namespace and those labels are only needed for cross-namespace network policies, so it should be fine.